src/wovnio/html/HtmlConverter.php

```diff
@@ -163,7 +164,7 @@ private function removeTagFromHtmlByRegex($html, $regex)
     return $result;
   }
 
-  private function buildSnippetCode()
+  private function buildSnippetCode($adds_backend_error_mark)
```

Comment:
I would prefer `is_backend_without_api` over `adds_backend_error_mark` as this is not error.

Reply:
?